src/app/views/interactives/InteractivesShow.test.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import mockAxios from "axios";
+import { render, screen } from "../../utilities/tests/test-utils";
+import InteractivesShow from "./InteractivesShow";
+import { show } from "../../utilities/api-clients/interactives-test";
+import * as reactRedux from "react-redux";
+
+const initialState = {
+    interactives: [],
+    interactive: {},
+    filteredInteractives: [],
+    errors: {
+        msg: {},
+    },
+    successMessage: {
+        type: null,
+        success: false,
+    },
+};
+
+describe("Delete an interactive", () => {
```

Comment:
unit tests should test user interactions not implementations. you should keep separations of concerns here